### ISSUE TITLE
Playlist autoplay toggle is not reachable via keyboard navigation

### ISSUE THREAD

**Issue by cjcolvar** [avalonmediasystem/avalon#3082]

**Comment by joncameron** [comment]
No longer an issue.
